playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.tsx

```diff
@@ -154,6 +154,6 @@ const Typeahead = ({
   )
 }
 
-export * as TypeaheadComponents from "./components"
+Typeahead.MenuList = MenuList
```

Comment:
To make this a "sub-component" style kit, you have to specify the assignments like so. It would be great if we could dynamically do this:

```diff
diff --git a/playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.tsx b/playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.tsx
index 1ecc4444..65758006 100644
--- a/playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.tsx
+++ b/playbook/app/pb_kits/playbook/pb_typeahead/_typeahead.tsx
@@ -18,6 +18,8 @@ import {
   ValueContainer,
 } from "./components"

+import * as kitComponents from "./components"
+
 import { noop, buildDataProps, buildHtmlProps } from '../utilities/props'
 import { GenericObject, Noop } from '../types'

@@ -154,6 +156,8 @@ const Typeahead = ({
   )
 }

-Typeahead.MenuList = MenuList
+Object.keys(kitComponents).forEach((k) => {
+  Typeahead.prototype[k] = (kitComponents as {[key: string]: unknown})[k]
+})

 export default Typeahead
 ```

Reply:
[This commit](https://github.com/powerhome/playbook/pull/3324/commits/e65b3fc9626dfbd52707d5496182ea53e09fcbe4) corrects the problem, but requires the `Typeahead` class to be cast to `GenericObject`. I'm not in love with this.

```tsx
Object.keys(kitComponents).forEach((k) => {
  (Typeahead as GenericObject)[k] = (kitComponents as {[key: string]: unknown})[k]
})
```